lib/send_with_us/file.rb

```diff
@@ -0,0 +1,16 @@
+module SendWithUs
```

Comment:
`SimpleDelegator` isn't part of the core library in some versions of ruby, so this could be a breaking change for some users. Can you add `require 'delegate'`? That should fix it.

Reply:
or just fully remove the dependency and add `#to_h` to File to call it over attachment, I forgot about old rubies (akka 1.8) and SimpleDelegator is super slow for ruby 1.8

Reply:
Agreed! This is way better. 😄